config/application.rb

```diff
@@ -19,5 +19,6 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+    config.action_view.field_error_proc = proc { |html_tag, _instance| html_tag.html_safe }
```

Comment:
Just curious what this is?

Reply:
Rails default validation does something really annoying, where it wraps invalid fields in a
```
<div class="field_with_errors">
  <your original field/>
</div>
```
This tells it to just output the field as is when there is an error for the field